picard/ui/options/cover_processing.py

```diff
@@ -114,7 +114,7 @@ def __init__(self, parent=None):
             checkbox.stateChanged.connect(self._update_resize_spinboxes)
 
     def _add_rich_text(self, text):
-        return "<FONT COLOR=black>" + text + "</FONT>"
+        return "<br>".join(text)
```

Comment:
From i18n point of view, instead of silently replacing newlines by HTML `<br />` it would be much better to make it explicit that's HTML-like rich text for translators.

```python
        tooltip_keep = N_(
            "<p>"
            "Scale the source image so that it fits within the target dimensions.<br />"
            "One of the final image dimensions may be less than the target dimension if "
            "the source image and target dimensions have different aspect ratios.<br />"
            "<br />"
            "For example, a 2000x1000 image resized to target dimensions of "
            "1000x1000 would result in a final image size of 1000x500."
            "</p>"
        )
        self.ui.tags_keep.setToolTip(_(tooltip_keep))
        self.ui.file_keep.setToolTip(_(tooltip_keep))
```

It gives something like:

![Capture d’écran du 2024-06-24 10-00-15](https://github.com/metabrainz/picard/assets/151042/78cb0ffd-a004-4ccd-ae2c-0cee11685cf0)


This is much better for languages not using left-to-right direction, as the translator can add `dir` attribute (depending on language):

Using `<p dir="rtl">` would give something like (with the same text):

![Capture d’écran du 2024-06-24 09-59-16](https://github.com/metabrainz/picard/assets/151042/998a1d6f-a084-40f7-8fce-2848155e366b)